<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSttBinRecordDlg</class>
 <widget class="QWidget" name="QSttBinRecordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>录波</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="m_HLayout_HeadBtns">
     <item>
      <widget class="QLabel" name="m_Label_SampleRate">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>采样率</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_Comb_SampleRate">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>30</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>4k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>35k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>45k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>55k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>65k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>70k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>75k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>85k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>90k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>95k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>110k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>120k</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="m_VLayout_ModulesGrid">
     <item>
      <widget class="QTableWidget" name="m_pBinRecordTopTableWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="m_VLayout_BinsGrid">
     <item>
      <widget class="QTableWidget" name="m_pBinRecordBottomTableWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="m_ChkAutoSaveComtradeFile">
       <property name="text">
        <string>自动保存录波文件</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_HLayOutBtns">
     <item>
      <widget class="QPushButton" name="m_Btn_RecordCfg">
       <property name="text">
        <string>录波设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_Btn_LoadCfg">
       <property name="text">
        <string>导入配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_Btn_SaveCfg">
       <property name="text">
        <string>保存配置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_Btn_BeginRecord">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>启动录波</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_Btn_StartMonitor">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>进入监视</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_Btn_Close">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
